src/ozone/ozone.py

```diff
@@ -482,11 +482,18 @@ def get_multiple_coordinate_air(
             selected another data format, this dataframe will be empty)
         """
         for loc in locations:
-            # This just makes sure that it's always a returns a pandas.DataFrame.
-            # Makes mypy happy.
-            df = pandas.DataFrame(
-                self.get_coordinate_air(loc[0], loc[1], df=df, params=params)
-            )
+            try:
+                # This just makes sure that it's always a returns a pandas.DataFrame.
+                # Makes mypy happy.
+                df = pandas.DataFrame(
+                    self.get_coordinate_air(loc[0], loc[1], df=df, params=params)
+                )
+            except Exception:
+                # NOTE: If we have custom exception we can catch it instead.
```

Comment:
Hmm overall I do think we should have custom exceptions.

However, it's a good idea to add custom exceptions for the entirety of Ozone in one go (I'm sure there are other sections that need them). This way we can come up with a consistent set of exceptions for the package.